tuna/worker_interface.py

```diff
@@ -134,6 +134,9 @@ def __init__(self, **kwargs):
     #also set cnx here in case WorkerInterface exec_command etc called directly
     self.cnx: Connection = self.machine.connect(chk_abort_file)
 
+  def get_miopen_v(self):
+    """Abstract method """
```

Comment:
should raise NotImplemented otherwise you are only writing this to pass pylint.

Reply:
This method needs to be removed from worker_interface.